maxim/logger/crewai/client.py

```
@@ -117,21 +117,23 @@ def make_handle_non_streaming_wrapper(original_method):
     """
 
     @functools.wraps(original_method)
-    def handle_non_streaming_wrapper(
-        self,
-        params: dict,
-        callbacks: Union[list, None] = None,
-        available_functions: Union[dict, None] = None,
-    ):
+    def handle_non_streaming_wrapper(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for private function `handle_non_streaming_wrapper`

(ANN202)